progress/angular/projects/ng-progress-icons/src/lib/pxb-progress-icon.scss

```diff
@@ -0,0 +1,15 @@
+.pxb-progress-icon {
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    display: inline-block;
+}
+.pxb-progress-icon-label {
+    text-align: center;
+    z-index: 2;
+    &.center {
+      font-weight: 900;
+      -webkit-text-fill-color: black;
+      -webkit-text-stroke-color: white;
```

Comment:
no pxblue colors for these?

Reply:
We don't have a dependency on PX Blue colors in this library. Technically it could be used in any project by anyone.

These do break the labelColor input though. Instead of hardcoding values for these, what if we use the `color` library to determine if the current `labelColor` is 'dark' or 'light' and then use black or white as the stroke. I believe the InfoListItem is doing something like this with the default icon color on top of the Avatar icon if you need a reference.

This looks awesome in Chrome, Safari, and Firefox...anybody dare to have a look at IE? I don't think we necessarily need to make it work in IE, but it would be good to know how broken it is.

Reply:
Good idea (tip my hat)

Reply:
Idk how you would use the color lib to detect an inherited color though

Reply:
This is a good idea, I'll add the `color` library as a peerDependency to this repo.  This will not fix the hardcoded values however, the black/white, white/black combination will still have to be an inline style.  

I think you can still override these values if you really need to using some css tricky, like this:
https://css-tricks.com/override-inline-styles-with-css/

Reply:
Just added in this logic but didn't push it out yet.  The options are down below.